manifests/frontend/load_balancer/website2.pp

```diff
@@ -73,23 +75,25 @@
       group   => 'sunetfrontend',
       mode    => '0640',
       force   => true,
-      content => inline_template("# File created from Hiera by Puppet\n<%= @config3.to_yaml %>\n"),
+      content => inline_template("# File created from Hiera by Puppet\n<%= @config4.to_yaml %>\n"),
       ;
   }
 
   # Parameters used in frontend/docker-compose_template.erb
+  $dns                    = pick_default($config['dns'], undef)
```

Comment:
Borde man inte sätta [] som default här - det blir en iteration över tomma listan senare som ju är "safe" liksom.

Reply:
Should be fixed in https://github.com/SUNET/puppet-sunet/pull/12/commits/e729ac21680d98ba6b2d888012959223e5b14467